internal/reports/report_helpers.go

```diff
@@ -383,40 +507,53 @@ func fillRowStats(conv *internal.Conv, srcTable string, badWrites map[string]int
+	internal.Widened:              {Brief: "Some columns will consume more storage in Spanner", severity: warning, batch: true, Category: "STORAGE_WARNING"},
+	internal.StringOverflow:       {Brief: "String overflow issue might occur as maximum supported length in Spanner is 2621440", severity: warning, Category: "STRING_OVERFLOW_WARNING"},
+	internal.HotspotTimestamp:     {Brief: "Timestamp Hotspot Occured", severity: warning, Category: "TIMESTAMP_HOTSPOT"},
+	internal.HotspotAutoIncrement: {Brief: "Autoincrement Hotspot Occured", severity: warning, Category: "AUTOINCREMENT_HOTSPOT"},
+	internal.InterleavedOrder: {Brief: "can be converted as Interleaved with Table", severity: suggestion, Category: "INTERLEAVE_TABLE_SUGGESTION",
+		CategoryDescription: "Some tables can be interleaved"},
+	internal.RedundantIndex:     {Brief: "Redundant Index", severity: warning, Category: "REDUNDANT_INDEX"},
+	internal.AutoIncrementIndex: {Brief: "Auto increment column in Index can create a Hotspot", severity: warning, Category: "AUTO-INCREMENT_INDEX"},
+	internal.InterleaveIndex: {Brief: "can be converted to an Interleave Index", severity: suggestion, Category: "INTERLEAVE_INDEX_SUGGESTION",
+		CategoryDescription: "Some columns can be interleaved"},
+	internal.InterleavedNotInOrder: {Brief: "if primary key order parameter is changed for the table", severity: suggestion, Category: "INTERLEAVED_NOT_IN_ORDER",
+		CategoryDescription: "Some tables can be interleaved with parent table if primary key order parameter is changed to 1"},
+	internal.InterleavedAddColumn: {Brief: "Candidate for Interleaved Table", severity: suggestion, Category: "ADD_INTERLEAVED_COLUMN",
+		CategoryDescription: "If there is some primary key added in table, it can be interleaved"},
+	internal.IllegalName: {Brief: "Names must adhere to the spanner regular expression {a-z|A-Z}[{a-z|A-Z|0-9|_}+]", severity: warning, Category: "ILLEGAL_NAME"},
+	internal.InterleavedRenameColumn: {Brief: "Candidate for Interleaved Table", severity: suggestion, Category: "RENAME_INTERLEAVED_COLUMN_PRIMARY_KEY",
+		CategoryDescription: "If primary key is renamed in table to match the foreign key, the table can be interleaved"},
+	internal.InterleavedChangeColumnSize: {Brief: "Candidate for Interleaved Table", severity: suggestion, Category: "CHANGE_INTERLEAVED_COLUMN_SIZE",
+		CategoryDescription: "If column size of this table's primary key is changed to match the foreign key, the table can be interleaved"},
+	internal.RowLimitExceeded: {Brief: "Non key columns exceed the spanner limit of 1600 MB. Please modify the column sizes", severity: errors, Category: "ROW_LIMIT_EXCEEDED"},
+	internal.ShardIdColumnAdded: {Brief: "column was added because this is a sharded migration and this column cannot be dropped", severity: note, Category: "SHARD_ID_COLUMN_ADDED",
+		CategoryDescription: "Some Column was added because this is a sharded migration and that column couldn't be dropped"},
```

Comment:
Shard id column was added because this is a sharded migration and this column cannot be dropped